tests/functional/setups/manu_flow_comp_2d_frac.py

```diff
@@ -660,19 +656,20 @@ def bc_type_mobrho(self, sd: pp.Grid) -> pp.BoundaryCondition:
             boundary_faces = self.domain_boundary_sides(sd).all_bf
             return pp.BoundaryCondition(sd, boundary_faces, "neu")
 
-    def bc_values_darcy(self, subdomains: list[pp.Grid]) -> pp.ad.DenseArray:
-        """Set boundary condition values for the elliptic discretization."""
-        bc_values = pp.ad.TimeDependentDenseArray(
-            name="darcy_bc_values", subdomains=subdomains, previous_timestep=True
-        )
-        return bc_values
+    def boundary_pressure(self, boundary_grid: pp.BoundaryGrid) -> np.ndarray:
```

Comment:
Missing docstring

Reply:
I added missing docstrings, so resolving this.